scripts/deploy/update_pay_server_docs.rb

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env ruby
+
+require 'subprocess'
+
+require_relative 'common'
+
+def update_pay_server_docs()
+    puts 'Ensuring pay-server repo is up-to-date.'
+    begin
+        execute_or_fail("git -C ../pay-server checkout master")
```

Comment:
Using master here because:
- I don't think we have a use case for deploying from a non-master branch of pay-server
- We can't reuse the "deploy branch" used within stripe-android, it doesn't necessarily exist within pay-server